playgame.py

```diff
@@ -67,33 +123,38 @@ def __init__(self, players, seed, variant):
         self.variant = variant
         self.current_player = 0
 
-    def play_game(self, verbose):
+    def play_game(self, verbose, logger):
```

Comment:
With how much `logger` is getting passed around here, would it make sense to make it class scoped, global, wrapped in a singleton class (and therefore global)?

Reply:
it used to be, and then I duplicated initialization once. Will make it global again